lib/hydra/works/services/characterization_service.rb

```diff
@@ -44,17 +44,11 @@ def fetch_or_respond(opts, key)
         opts.fetch(key) { object.send(key) if object.respond_to? key }
       end
 
-      # @param [String,Symbol,File]
-      # @return content if source is a symbol, File if source is string
+      # @return content of object if source is nil; otherwise, return a File or the source
       def source_to_content
-        if source.is_a? String
-          File.open(source)
-        elsif source.is_a? Symbol
-          s = object.send(source)
-          s.respond_to?(:content) ? s.content : s
-        else
-          source
-        end
+        return object.content if source.nil?
```

Comment:
We'll only be calling characterization on pcdm:files now, correct?  So the technical metadata is going on the pcdm:file that currently has the content, or will end up with the content?